### DISCUSSION TITLE
Bandwidth problems with bigger files?

### DISCUSSION THREAD

**Discussion by gklasen** [hgouveia/node-downloader-helper#49] [Q&A]
Hey there, first of all thank for for this nice plugin. I have it installed within an node.js Electron application.

I have the problem, that if i am trying to download bigger videos (like >20 MB maybe?) the plugin starts nicely, but very quickly, performance drops down. It starts with 7MB/s, but directly goes down to 1MB/s

Similar behaviour occurs with npm request plugin, but works flawlessly as plain browser download. Any chance to solve this by different config etc?

If not, maybe its solvable via "hack": Funnily If i pause the download every 3 seconds and resume it back, it quickly downloads 7MB-bunches step by step, but unfortunately, output video is malformed afterwards.

**Comment by Chaphasilor** [comment]
Yeah that's a common behavior. Could be that Node.js's http implementation is not a robust as it could be. Maybe your browser uses threaded downloads, thereby "emulating" the stopping and resuming. If this is the case then it might even be the server's fault, limiting the throughput after a few seconds, it's not unheard of...
